common/stringtable.cpp

```diff
@@ -338,7 +341,7 @@ bool StringTable::hasString(const OString& name) const
 //
 // Load strings from all LANGUAGE lumps in all loaded WAD files.
 //
-void StringTable::loadStrings()
+void StringTable::loadStrings(const bool engOnly)
```

Comment:
Just a question here - what's the purpose of having a boolean for english only?  That is, what would pass in a true to engOnly?